guide/daily-spending-wallet/sending.md

```diff
@@ -165,7 +165,7 @@ When responding to an invoice that contains all relevent information, the user c
 
 **Recipient**
 
-The most convenient option for choosing a recipient is from previously saved contacts. Alternatively, users can enter on-chain addresses, Lightning addresses, Lightning node IDs, or other static identifies that are supported by the wallet.
+The most convenient option for choosing a recipient is from previously saved [contacts]({{ '/guide/daily-spending-wallet/contacts/' | relative_url }}). Alternatively, users can enter Lightning addresses, Lightning node IDs, on-chain addresses, or other static identifies that are supported by the wallet.
 
 There are also static [invoice types]({{ '/guide/how-it-works/payment-request-formats/' | relative_url }}) that can receive payments repeatedly. These are less intuitive overall due to their appearance, but could also be considered payment endpoints.
```

Comment:
```suggestion
There are also static [payment requests]({{ '/guide/how-it-works/payment-request-formats/' | relative_url }}), which in the guide we call addresses, that can receive payments repeatedly. These are less intuitive overall due to their appearance, but could also be considered payment endpoints.
```

Reply:
Not sure what you mean with the second addition. `Address` is a user-facing term to abstract away all the various payment request formats, not just static ones. It's not what we call payment requests in the guide in general. Could you please clarify?

Reply:
Based on reading the contacts `Address` is only applicable to the reusable payment requests? Why use a user-facing term in the contacts page but not here?

Reply:
Sorry, my point is more that we should clarify that it's a user-facing term.

`which in the guide we call addresses`
Makes me think that `address` is the term used in all pages in the guide.

`which we simplify under the user-facing term "addresses" in this reference design`
Makes it clear that the term is for users, and only applicable in the daily spending wallet. No clue if the savings wallet or merchant section will tackle this differently.

Reply:
Okay that makes sense, edited my comment!